addons/mail/static/src/core/common/thread_model.js

```diff
@@ -188,6 +188,8 @@ export class Thread extends Record {
     followersCount;
     loadOlder = false;
     loadNewer = false;
+    /** @type {string} */
+    userSignature;
```

Comment:
Also we're now trying to just have the same field name in JS as we have in python, so just `signature`, which makes more sense as a simple name if the field is moved to persona.